tests/monitoring/monitoring.go

```diff
@@ -0,0 +1,143 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package monitoring
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"kubevirt.io/kubevirt/tests"
+
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/tests/flags"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	virtOperatorDeploymentName = "virt-operator"
+)
+
+var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
+
+	var err error
+	var virtClient kubecli.KubevirtClient
+	var scales = make(map[string]*autoscalingv1.Scale, 1)
```

Comment:
Shouldn't this get initialized in the `BeforeEach` to avoid interference between the tests (I see that there is only one test ATM, but I assume that more tests will follow later on)?

Reply:
I moved the initialization into BeforeEach function